### PR TITLE
Checkpoint id is undefined when checkpoint creation fails

### LINKED ISSUE #298: Undefined variable
bildschirmfoto von 2016-04-06 22-14-37

### PR DESCRIPTION
**Pull request by VicDeo**
Fixes #298

### CODE REVIEW

File: src/Command/CheckpointCommand.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ protected function execute(InputInterface $input, OutputInterface $output){
 				$checkpointId = $checkpoint->create();
 				$output->writeln('Created checkpoint ' . $checkpointId);
 			} catch (\Exception $e){
-				$output->writeln('Error while creating a checkpoint ' . $checkpointId);
+				$output->writeln('Error while creating a checkpoint ');
```
Comment (DeepDiver1975) [inline src/Command/CheckpointCommand.php]: what to do with the exception?
please remove the trailing whitespace as well

Comment (DeepDiver1975) [inline src/Command/CheckpointCommand.php]: > what to do with the exception?

@VicDeo

**Comment by PVince81** [comment]
:+1: code looks good

**Comment by PVince81** [comment]
@DeepDiver1975 second review ? (you reported this)

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**